### DISCUSSION TITLE
Git: From https://github.com/mygithubaccname/portfolio

### DISCUSSION THREAD

**Discussion by jnlliee** [microsoft/SATechnicalOnboarding#53] [Questions]
I can't sync changes and can't deploy web application. it says "create static web app portfolio succeeded" but it doesn't actually open it. And there is no "Open Action in GitHub" button. It just gives an error Git: From https://github.com/mygithubaccname/portfolio. how can I fix that? 
<img width="1440" alt="Screen Shot 2023-08-03 at 6 39 02 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/104841981/80f9e9bb-ae30-413f-97b2-aa3b8e3f3e4c">

**Comment by ManuSquall** [comment]
Hello @jnlliee , have you setup your global git user name and user email?

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/29026887/e64ac671-a6dc-4153-b3dd-c96be925c20e)

**Reply by jnlliee** [reply]
No, I haven't! Should I paste it the same way?

**Reply by ManuSquall** [reply]
Try it in the terminal by replacing with your own "name" and "email" according to your github account

**Comment by jnlliee** [comment]
okay, I will try! thank you so much

**Comment by jnlliee** [comment]
I did try it in the terminal with my  name and email, but it still doesn't work(
<img width="1440" alt="Screen Shot 2023-08-03 at 8 49 44 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/104841981/3a1856d9-c4a4-45d8-9333-41045e02d4b3">

**Reply by ManuSquall** [reply]
Ok, going to check it on my side and come back to you
